jujugui/static/gui/src/app/components/modal/modal.js

```diff
@@ -0,0 +1,44 @@
+/* Copyright (C) 2015 Canonical Ltd. */
+
+'use strict';
+
+class Modal extends React.Component {
```

Comment:
Why not use the existing `Panel` component? Also if this is required, it's missing a test file :)